src/test/resources/Room.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Confirming the functionality of the room top column
+
+  Scenario Outline: Validating the functionality of 'Room' column
+    Given The user has logged in the website
+    When The user clicks on the "Rooms" top column
+    And The user clicks on the Click to Select Dates tab
+    Then User validates calendar is present
+
+    When User selects the start date and end date
```

Comment:
this step must take in parameters so we can change the dates on the feature file

Reply:
Done